packages/catalog-readmodel-writer/src/model/utils.ts

```diff
@@ -0,0 +1,9 @@
+// Temporary workaround
```

Comment:
Maybe this comment is not needed, it's enough to have the comments in the other file where this function is used